inference-engine/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp

```diff
@@ -76,56 +110,94 @@ namespace {
 std::vector<fusingSpecificParams> fusingParamsSet {
         emptyFusingSpec,
         fusingMultiplyPerTensor,
-        fusingMultiplyPerChannel,
-        fusingAddPerTensor,
-        fusingAddPerChannel,
-        fusingSubtractPerTensor,
-        fusingSubtractPerChannel,
-        fusingDividePerTensor,
-        fusingDividePerChannel,
-        fusingPReluPerChannel,
-        fusingPReluPerTensor,
         fusingRelu,
-        fusingGelu,
-        fusingReluScaleShift
+        fusingPReluPerChannel
+};
+
+std::vector<fusingSpecificParams> fusingParamsSetDynamic {
```

Comment:
what about per channel post ops for dynamic shapes cases?

Reply:
Each ND case with dynamic case has separate instantiate with per channels post op. For example [2d](https://github.com/openvinotoolkit/openvino/blob/25afb7888ae6662aafe0987b76462db33f33b4b7/inference-engine/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp#L174)